mkdocs.yml

```diff
@@ -87,6 +92,23 @@ extra:
     link: https://discord.tacticalteam.de
     name: TTT-Discord-Server
 
+  - icon: fontawesome/brands/steam
+    link: https://steamcommunity.com/groups/tacticaltrainingteam
+    name: TTT-Steam-Gruppe
+
+  - icon: fontawesome/brands/youtube
+    link: https://www.youtube.com/channel/UCUywnzXGth9-iSqR2f6hP7g
+    name: TTT-YouTube-Kanal
+
+  - icon: fontawesome/solid/gavel
+    link: https://tacticalteam.de/datenschutz
+    name: Datenschutzerklärung
+
+  analytics:
+    #Matomo
+    provider: custom
+    property: https://analytics.tacticalteam.de/
+
```

Comment:
Das ist nur für die integrierte Google Analytics relevant. Da sparen wir uns die variablen- also kann weg, wenn das JS von Matomo komplett in die neue Datei kommt